uni/lib/view/Pages/exams_page_view.dart

```diff
@@ -136,13 +146,9 @@ class ExamsList extends StatelessWidget {
                 ? Theme.of(context).hintColor
                 : Theme.of(context).scaffoldBackgroundColor,
             child: ScheduleRow(
-              subject: exam.subject,
-              rooms: exam.rooms,
-              begin: exam.begin,
-              end: exam.end,
-              type: exam.examType,
-              date: exam.date,
-              teacher: '',
+              exam: exam,
+              exams: hidden,
+              mainPage: false,
```

Comment:
We could calculate whether the exam is hidden or not in this function, and change the ScheduleRow's List _exams_ to a boolean _isHidden_

Reply:
I undestand why you passed the List containing the hidden exams, to change them. However it isn't a good pratice to pass this kind of list to a single element. Below I suggested an alternative (could be better), however if you find something better I would appreciate it. This could be discussed